python/helpers/pydev/pydevd_file_utils.py

```diff
@@ -73,10 +73,29 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+def convert_to_long_pathname(filename):
```

Comment:
Ok, you're right. Let's move ctypes checking out of the function on the top level, save results of the checking to the variable and use it inside function.